bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemRegistry.java

```diff
@@ -25,6 +25,14 @@
 public interface ItemRegistry extends Registry<Item, String> {
 
     /**
+     * This methods checks if an item is part of the registry.
+     *
+     * @param name the item name
+     * @return true if there is an item with given name, otherwise false
+     */
+    public boolean containsItem(String name);
```

Comment:
Not sure we should touch the ItemRegistry interface here - a 'get(name)!=null' would do as well. If we feel we need an "exists" method, we should imho rather add it as 'Registry.exists(K)'.